rocky/katalogus/templates/katalogus.html

```diff
@@ -50,7 +50,7 @@ <h1>{% translate "KAT-alogus" %}</h1>
                             </thead>
                             <tbody>
                                 {% for plugin in object_list %}
-                                    <tr>
+                                    <tr id="enableform_{{ plugin.id|slugify }}">
```

Comment:
Should we still name the id enableform if it is put on the row instead of the form?

Reply:
Indeed, it doesn't make sense to use that prefix anymore. E.g. `plugin_` should suffice, but it also makes sense to not use any prefix

Reply:
agreed, we should call it enableplugin instead.

Reply:
Yes, it was a bit lazy to just move the ID and not rethink its prefix. Just fixed it and altered it to "plugin_"